runtime/gc_structs/PointerContiguousArrayIterator.hpp

```diff
@@ -108,8 +111,11 @@ class GC_PointerContiguousArrayIterator
 	 * @param objectPtr the array object to be processed
 	 */
 	GC_PointerContiguousArrayIterator(OMR_VM *omrVM, J9Object *objectPtr)
-		: _slotObject(GC_SlotObject(omrVM, NULL))
-#if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
+		:	_arrayPtr(NULL)
+		,	_slotObject(GC_SlotObject(omrVM, NULL))
+		,	_scanPtr(NULL)
+		,	_endPtr(NULL)
+if defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS)
 		, _compressObjectReferences(OMRVM_COMPRESS_OBJECT_REFERENCES(omrVM))
 #endif /* defined(OMR_GC_COMPRESSED_POINTERS) && defined(OMR_GC_FULL_POINTERS) */
 		, _omrVM(omrVM)
```

Comment:
we can do:

	GC_PointerContiguousArrayIterator(OMR_VM *omrVM, J9Object *objectPtr)
		: GC_PointerContiguousArrayIterator(omrVM)
	{
		initialize(objectPtr);
	}

Reply:
And even expand the change and update it for GC_PointerArrayletIterator and GC_MixedObjectIterator.

It's more future proof (less likely that someone will change one and miss the other).